rasa_core/conversation.py

```diff
@@ -17,9 +17,34 @@ class Dialogue(object):
     def __init__(self, name, events):
         # type: (Text, List[Event]) -> None
 
+        # This function initialises the dialogue with
+        # the dialogue name and the event list.
         self.name = name
         self.events = events
 
     def __str__(self):
+        # type: None -> Text
+
+        # This function returns the dialogue and turns.
         return "Dialogue with name '{}' and turns:\n{}".format(
                 self.name, "\n\n".join(["\t{}".format(t) for t in self.events]))
+
+
+class Topic(object):
```

Comment:
This class got removed on master and should not be readded. (this includes the code below till the end of the file)